tests/TestCase/ORM/EntityTest.php

```diff
@@ -69,6 +69,13 @@ public function testSetMultiplePropertiesNoSetters()
         $this->assertSame(3, $entity->thing);
         $this->assertSame('bar', $entity->getOriginal('foo'));
         $this->assertEquals(1, $entity->getOriginal('id'));
+
+        $entity->set(['foo', 'bar']);
+        $this->assertEquals('foo', $entity->{0});
```

Comment:
Better to use `$entity[0]` or `$entity->get('0')` instead of `$entity->{0}`.

Reply:
I fixed it.